### PR TITLE
Migrate over to use Qubit API

### PR DESCRIPTION
**Pull request by alexolivier**
- [x] GraphQL Query
- [x] Map back to original output for backward compatibility
- [x] Support rules type
- [x] Update tests
- [x] Use `QubitAPI` package

### CODE REVIEW

**Comment by alexolivier** [comment]
@roberttod can you give this a once over again, please? I switched over to use the `qubit-api` package

File: get.js [hunk 1]
```diff
@@ -1,7 +1,42 @@
 var _ = require('slapdash')
-var http = require('@qubit/http-api')
+var qubitApi = require('@qubit/qubit-api')
 var getLocale = require('./getLocale')
 
+var query = [
```
Comment (roberttod) [inline get.js]: Can use es6, so use something like
```js
const query = `
  query () {
    stuff ()
  }
`
```

File: get.js [hunk 2]
```diff
@@ -17,51 +52,70 @@ module.exports = function getRecommendations (config, options) {
...
-          var items = _.get(recs, 'result.items')
-
+          var items = _.get(
+            result,
+            'data.property.visitor.productRecommendations'
+          )
           if (items && items.length) {
+            // Convert back to native Recs API format to support legacy experiences
+            items = _.map(items, function (item) {
+              item.id = item.product.product_id
```
Comment (roberttod) [inline get.js]: Can we use an empty object to return from this function, rather than mutating these items.

File: get.js (hunk 1 above)
Comment (alexolivier) [reply get.js]: Nice didn't know we supported that in packages - guess the move to webpack sorts that out. Shall update

**Pull request merged**